src/communitiesMetadata.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "groupIds": [
```

Comment:
we should probably load these on the fly from `/v3/groups/` endpoint and then do some janky matching on names to pair up the subdomains.

Reply:
v3 or v5?

Reply:
@ThomasKranitsas `v5/groups` is still in development and can't be used. So we need to use `v3` for now.

cc: @cwdcwd